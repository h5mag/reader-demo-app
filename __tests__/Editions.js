import React from 'react';
import renderer from 'react-test-renderer';
import EditionReader from '../src/components/EditionReader';

test('renders correctly', () => {
  const tree = renderer.create(<EditionReader />).toJSON();
  expect(tree).toMatchSnapshot();
});
