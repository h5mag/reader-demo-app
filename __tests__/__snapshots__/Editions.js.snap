// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "flex": 1,
          "overflow": "hidden",
        },
        undefined,
      ]
    }
  >
    <RNCWebView
      allowFileAccess={true}
      allowFileAccessFromFileURLs={true}
      allowUniversalAccessFromFileURLs={true}
      cacheEnabled={false}
      domStorageEnabled={true}
      injectedJavaScriptBeforeContentLoadedForMainFrameOnly={true}
      injectedJavaScriptForMainFrameOnly={true}
      javaScriptEnabled={true}
      messagingEnabled={false}
      onContentProcessDidTerminate={[Function]}
      onHttpError={[Function]}
      onLoadingError={[Function]}
      onLoadingFinish={[Function]}
      onLoadingProgress={[Function]}
      onLoadingStart={[Function]}
      onMessage={[Function]}
      onShouldStartLoadWithRequest={[Function]}
      source={
        Object {
          "uri": undefined,
        }
      }
      style={
        Array [
          Object {
            "flex": 1,
            "overflow": "hidden",
          },
          Object {
            "backgroundColor": "#ffffff",
          },
          Object {
            "backgroundColor": "black",
            "flex": 1,
          },
        ]
      }
      textInteractionEnabled={true}
      useSharedProcessPool={true}
    />
  </View>
</RCTSafeAreaView>
`;
